ios/RNGestureHandlerManager.mm

```diff
@@ -211,8 +213,14 @@ - (void)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
     // Once the upstream fix lands the line below along with this comment can be removed
     if ([gestureRecognizer.view isKindOfClass:[UIScrollView class]]) return;
 
+#ifdef RN_FABRIC_ENABLED
+    RCTSurfaceTouchHandler *touchHandler = [viewWithTouchHandler performSelector:@selector(touchHandler)];
+    [touchHandler setEnabled:NO];
+    [touchHandler setEnabled:YES];
+#else
     RCTTouchHandler *touchHandler = [viewWithTouchHandler performSelector:@selector(touchHandler)];
     [touchHandler cancel];
+#endif
```

Comment:
```suggestion
#ifdef RN_FABRIC_ENABLED
    RCTSurfaceTouchHandler *touchHandler = [viewWithTouchHandler performSelector:@selector(touchHandler)];
#else
    RCTTouchHandler *touchHandler = [viewWithTouchHandler performSelector:@selector(touchHandler)];
#endif
    // Cancel the touchHandler
    [touchHandler setEnabled:NO];
    [touchHandler setEnabled:YES];
```

Since `[touchHandler cancel]` does exactly the same thing, wouldn't it be cleaner this way to show that the same action is taken regardless of the `touchHandler`? (This is more of a question than a suggestion)